IPython/html/static/notebook/less/notebook.less

```diff
@@ -5,6 +5,32 @@
         padding-right: 0px;
     }
 }
+/* Sticky header effect. Includes media queries to keep 
+   margin between #site and #header when resizing. 
+ */
+
+.header_notebook {
+    position: fixed;
+    width: 100%;
+}
+
+.site_notebook {
+    margin-top: 9em;
+}
+
+@media screen and (-webkit-min-device-pixel-ratio:0) and (max-width: 636px) {
+      .site_notebook {
+          margin-top: 11em;
+        }
+}
+
+@media (max-width: 633px) {
```

Comment:
Well, the same issue that happens when we resize the window and the height of the header changes. Any change in the header's height is somewhat awkward to deal with but fixable. The issue is finding the most elegant solution in this case. I think @minrk's suggestion of using `calc` is quite possible but I wanted to revisit the percentage idea again, because that is faster and doesn't depend on features that are still considered experimental/subject to change.